wrapanapi/systems/virtualcenter.py

```diff
@@ -243,10 +284,132 @@ def _get_resource_pool(self, resource_pool_name=None):
             return self.system.get_obj(vim.ResourcePool, self.system.default_resource_pool)
         return self.system.get_obj_list(vim.ResourcePool)[0]
 
-    def _clone(self, destination, resourcepool=None, datastore=None, power_on=True,
-               sparse=False, template=False, provision_timeout=1800, progress_callback=None,
+    def _get_cluster_compute_resource(self, cluster_compute_resource_name=None):
```

Comment:
Q: why the wordy names? can this just be `_get_cluster`? or `cluster_name`?

Reply:
So for this method, we are getting the cluster computer resource. So I wanted to be descriptive in the name. Here it is more than just a name:
https://www.vmware.com/support/developer/converter-sdk/conv55_apireference/vim.ClusterComputeResource.html